index.html

```diff
@@ -0,0 +1,130 @@
+      <nav class="nav-desktop">
+        <a href="./index.html">
+          <img src="assets/icons/Logo.svg" alt="Logo" class="index-logo" />
+        </a>
+        <ul class="nav-desktop-ul">
+          <li class="nav-desktop-li"><a href="./index.html">HOME</a></li>
+          <li class="nav-desktop-li"><a href="/Products">PRODUCTS</a></li>
+          <li class="nav-desktop-li"><a href="/Sponsor">PARTNERS</a></li>
+          <li class="nav-desktop-li"><a href="/News">NEWS</a></li>
+          <li class="nav-desktop-li">
+            <a href="pages/about.html">ABOUT</a>
+          </li>
+          <li>
+            <a href="#" class="ki-campaign">KI Internship</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="hero">
+      <h3 class="hero-text-1">Greetings! welcome to</h3>
+      <h1 class="hero-header-1 mt-20">KRAMERICA INDUSTRIES</h1>
+      <p class="hero-text-2 mt-20">
+        It's business as usual here at Kramerica Industries, we are a space
+        value creation company that provides people with inspiring and exciting
+        experiences by fusing digital technology and real space. Lorem ipsum,
+        dolor sit amet consectetur adipisicing elit. Totam explicabo libero
+        neque cupiditate est dolore earum repellat illo voluptatem rerum ea, ab
+        suscipit animi, voluptates incidunt! Laboriosam nulla vel nostrum?
+      </p>
+      <p class="hero-text-3 mt-20">
+        Launching Our Oil Bladder System <br />
+        @ Burj Khalifa
+      </p>
+      <h2 class="hero-header-2 mt-20">2022.03.30(THUR) ~ 09.16</h2>
+    </main>
+
+    <section class="services mt-20">
+      <h2 class="services-heading">Our Services</h2>
+      <ul id="services-ul"></ul>
+      <a href="#" class="services-button mt-20">Need our products?</a>
+    </section>
+
+    <section class="employees mt-20">
+      <h2 class="employees-heading">Our Employees</h2>
+      <ul id="employees-ul" class="mt-20"></ul>
+      <button type="button" class="employees-button mt-20">
+        More <img src="assets/icons/more.svg" alt="load more employees" />
+      </button>
+    </section>
+
+    <section class="index-partners mt-20">
+      <h2 class="index-partners-heading">Partner</h2>
+      <ul class="mt-20">
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-41.svg" alt="vandelay" />
+        </li>
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-6.svg" alt="bluth.png" />
+        </li>
+        <li class="index-partners-item">
+          <img
+            src="assets/icons/logoipsum-logo-25.svg"
+            alt="dundermifflin.jpg"
+          />
+        </li>
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-8.svg" alt="fightmilk.png" />
+        </li>
+        <li class="index-partners-item">
+          <img src="assets/icons/logoipsum-logo-53.svg" alt="lospolos.png" />
+        </li>
+      </ul>
```

Comment:
- [ ] Please make sure to remove the dots from your list elements in this section, so you can stick to the given design as much as possible.
- [ ] Also, still regarding these elements, try to follow the design for displaying them in the screen. If you check now, you will have all your elements in a column and some extra white space on the right side, please work on it.
Here's an image to reference
![Captura de pantalla 2022-03-10 211036](https://user-images.githubusercontent.com/59100477/157746534-2b4bc109-c6a5-40b9-9602-cbd7b99a09ac.jpg)
- [ ] Last but not least, if you check the given design, you will realize that this section has a dark background which you are not giving to your project, so please try to work on that as well.

Reply:
Hey @Lordkaito, Every issue besides this particular one has been fixed, this seems to be a problem from your end. Everything renders perfectly over here.